app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Tuto4U</string>
+    <string name="user_name_text">Username</string>
```

Comment:
use hypen instead of underscore

Reply:
In Android it is a standard to use underscores to separate string resource key name words. Keeping it as it is for now. Let us know your thoughts. Thank you.